buildenv/jenkins/openjdk_x86-64_windows

```diff
@@ -0,0 +1,18 @@
+#!groovy
+LABEL='hw.arch.x86&&sw.os.win'
```

Comment:
Should we use `sw.os.windows` according to https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/93?
Eclipse OpenJ9 is already updated to use `sw.os.windows`

Reply:
Yes we should use sw.os.windows, I will update and merge.  Thanks!